src/create_update.c

```diff
@@ -387,8 +387,10 @@ int main(int argc, char **argv)
 		name_includes = manifest->includes;
 		while (name_includes) {
 			char *name = name_includes->data;
+			gpointer tmp = g_hash_table_lookup(new_manifests, name);
```

Comment:
Not keen on throwing away types here in the name of glibisms.

gpointer -> void *

For the sake of clarity surely this should be typed to the value type expected